pkg/authorization/bearer_token.go

```diff
@@ -0,0 +1,75 @@
+package authorization
+
+import (
+	"crypto/rand"
+	"fmt"
+	"os"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+const (
+	TokenPathEnvVar = "KUBERNETES_SERVICE_TOKEN_PATH"
+
+	defaultTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+)
+
+var inMemoryToken *[]byte
```

Comment:
no need to use a pointer to slice. non initiated slice is already nil.

Reply:
done